apps/system/js/notifications.js

```diff
@@ -363,6 +367,17 @@ var NotificationScreen = {
     return date;
   },
 
+  updateNotificationsDir: function ns_updateNotificationsDir() {
+    var newDir = document.documentElement.dir;
+    var notificationGroup = document.getElementsByClassName('notification');
+    for (var i = 0, l = notificationGroup.length; i < l; i++) {
+      var predefinedDir = notificationGroup[i].dataset.predefinedDir;
+      if ((predefinedDir === 'auto') || !predefinedDir) {
```

Comment:
Let's have a similar explanation as below as to why we don't want to use the 'auto' dir if it was defined by the notification.